changelog/unreleased/issue-323

```diff
@@ -0,0 +1,11 @@
+Enhancement: Add a command to copy a snapshot to another repository
```

Comment:
Suggest changing to `Enhancement: Add command to copy snapshots between repositories`.

Reply:
Is changed.